src/coord/src/coord.rs

```diff
@@ -6176,7 +6535,7 @@ mod timeline {
         ///
         /// If `lower_bound` is strictly greater than the current time (of either state), the
         /// resulting state will be `Writing(lower_bound)`.
-        pub fn fast_forward(&mut self, lower_bound: T) {
+        pub fn _fast_forward(&mut self, lower_bound: T) {
```

Comment:
Should we remove this method or keep it around?

Reply:
If it's not called anywhere, remove it.